service/src/main/java/bio/terra/workspace/common/logging/WorkspaceActivityLogHook.java

```diff
@@ -353,30 +355,28 @@ private void maybeLogForSyncGcpIamRolesFlight(
     }
   }
 
-  private void maybeLogUpdateControlledResourceRegionFlight(
+  private void maybeLogUpdateControlledResourceFieldsFlight(
       FlightContext context, OperationType operationType, String userEmail, String subjectId) {
     if (!context.getFlightStatus().equals(FlightStatus.SUCCESS)) {
       return;
     }
     FlightUtils.validateRequiredEntries(
-        context.getWorkingMap(),
-        CONTROLLED_RESOURCE_ID_TO_REGION_MAP,
-        CONTROLLED_RESOURCE_ID_TO_WORKSPACE_ID_MAP);
-    Map<UUID, String> resourceIdToRegionMap =
-        context.getWorkingMap().get(CONTROLLED_RESOURCE_ID_TO_REGION_MAP, new TypeReference<>() {});
+        context.getWorkingMap(), CONTROLLED_RESOURCE_ID_TO_WORKSPACE_ID_MAP);
+
```

Comment:
Why is the RESOURCE_ID_TO_REGION_MAP no longer needed?

Reply:
The `maybeLogUpdateControlledResourceFieldsFlight` logs the GCP region, Azure region, and the BQ lifetime backfill update flights.

Before, the code was specific to the two region flights. Now, it's generalized to include the BQ lifetime flight (see Yu's comment from earlier: https://github.com/DataBiosphere/terra-workspace-manager/pull/1041#discussion_r1104640810).